src/data/sym/basic.lean

```diff
@@ -225,9 +225,11 @@ end
 lemma repeat_left_inj (a b : α) (n : ℕ) (h : n ≠ 0) : repeat a n = repeat b n ↔ a = b :=
 (repeat_left_injective n h).eq_iff
 
-instance nontrivial (n : ℕ) [nontrivial α] : nontrivial (sym α (n + 1)) :=
+instance (n : ℕ) [nontrivial α] : nontrivial (sym α (n + 1)) :=
 (repeat_left_injective n.succ n.succ_ne_zero).nontrivial
 
+/-- The multiplicity of `b` in `map f s` is the number of `a ∈ s` (counting multiplicity)
+  such that `f a = b`. -/
```

Comment:
This is pretty obscure. You can just say
```suggestion
/-- A map `f : α → β` induces the pointwise map `sym2.map h : sym α n → sym β n`. -/
```

Reply:
I'm not sure about "pointwise" since `sym` isn't really a pi type, so I'll throw in another suggestion:
```suggestion
/-- A function `α → β` induces a function `sym α n → sym β n` by applying it to every element of the underlying `n`-tuple. -/
````